### PR TITLE
Add pre-commit and Travis

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVP-1177

### CODE REVIEW

File: kirin/abstract_sncf_model_maker.py [hunk 1]
```diff
@@ -41,20 +41,20 @@
...
-TRAIN_ID_FORMAT = 'OCE:SN:{}'
+TRAIN_ID_FORMAT = "OCE:SN:{}"
 SNCF_SEARCH_MARGIN = timedelta(hours=1)
 
 
 class TripStatus(Enum):
-    AJOUTEE = 1,    # Added
-    SUPPRIMEE = 2,  # Deleted
-    PERTURBEE = 3   # Modified or Impacted
+    AJOUTEE = (1,)  # Added
```
Comment (nonifier) [inline kirin/abstract_sncf_model_maker.py]: Out of curiosity, what does this syntax mean ? `(1,)`

Comment (nonifier) [reply kirin/abstract_sncf_model_maker.py]: Should an enum not be written without the trailing `,` ?

Comment (mbounabi) [reply kirin/abstract_sncf_model_maker.py]: I guess that one member of the pre-commit party did this...

**Comment by mbounabi** [comment]
Jenkins, test this please.

**Pull request merged**